charts/aad-pod-identity/templates/infra.yaml

```diff
@@ -146,9 +147,10 @@ spec:
       {{- end }}
       containers:
       - name: mic
-        image: "mcr.microsoft.com/k8s/aad-pod-identity/mic:1.2"
+        image: "mcr.microsoft.com/k8s/aad-pod-identity/mic:1.3"
         imagePullPolicy: Always
         args:
+          - mic
```

Comment:
What does name as parameter do ?

Reply:
we moved to alpine images with different docker file. This is just directly calling mic as container entry point (this is across mic/nmi on charts and deployment spec)